### PR TITLE
port remove signature on MAC host

### PR DESCRIPTION
**Pull request by wli3**
Part of https://github.com/dotnet/sdk/issues/3773

@swaroop-sridhar

### CODE REVIEW

**Review by swaroop-sridhar — APPROVED** [review]

**Review by nguerrera — APPROVED** [review]

**Comment by wli3** [comment]
get @leecow 's approval. Will merge once tests passed

**Pull request merged**
